ui/app/AppLayouts/Profile/views/WalletView.qml

```diff
@@ -15,61 +17,77 @@ import "../popups"
 import "../panels"
 import "./wallet"
 
-ScrollView {
+Item {
     id: root
 
     property var emojiPopup
     property WalletStore walletStore
 
-    clip: true
+    anchors.fill: parent
 
-    StackLayout {
-        id: stackContainer
+    StatusBanner {
+        id: banner
+        anchors.left: parent.left
+        anchors.top: parent.top
+        anchors.right: parent.right
+        visible: walletStore.areTestNetworksEnabled
+        type: StatusBanner.Type.Danger
+        statusText: qsTr("Testnet mode is enabled. All balances, transactions and dApp interactions will be on testnets.")
+    }
 
-        anchors.fill: parent
-        currentIndex: 0
+    ScrollView {
 
-        MainView {
-            id: main
-            Layout.preferredWidth: 560
-            leftPadding: 64
-            topPadding: 64
-            walletStore: root.walletStore
+        anchors.top: banner.visible ? banner.bottom: parent.top
+        clip: true 
 
-            onGoToNetworksView: {
-                stackContainer.currentIndex = 1
-            }
+        StackLayout {
+            id: stackContainer
 
-            onGoToAccountView: {
-                root.walletStore.switchAccountByAddress(address)
-                stackContainer.currentIndex = 2
-            }
+            anchors.fill: parent
+            currentIndex: 0
+
+            MainView {
+                id: main
+                Layout.preferredWidth: 560
+                leftPadding: 64
+                topPadding: 64
+                walletStore: root.walletStore
 
-            onGoToDappPermissionsView: {
-                stackContainer.currentIndex = 3
+                onGoToNetworksView: {
+                    stackContainer.currentIndex = 1
```

Comment:
Maybe constants would help to understand where we are switching and also limit the issues if we ever switch the order or add new views in the stack

Reply:
Good point, updated